.projenrc.js

```diff
@@ -25,6 +25,8 @@ const project = new awscdk.AwsCdkConstructLibrary({
     'fs-extra',
     'indent-string',
     'micromatch',
+    'next',
```

Comment:
this might make things way way bigger and I don't think it's needed?

Reply:
I was getting red squiggly lines in my editor. Is there a way to include this as a devDep? I can't edit my package.json b/c it's autogenerated from the projen.
Can I add it to the `devDeps` array in `.projenrc.js`?

Reply:
maybe